src/openfga.ne

```diff
@@ -39,23 +89,38 @@ define          ->  _newline define_initial (_spacing | _relation_types) _as _sp
                 type: 'single',
                 targets: [def]
             }
-        let allowedTypes = data[2] ? data[2][0] : [];
-        if (allowedTypes.length && typeof allowedTypes[0] !== "string") {
-          allowedTypes = [];
-        }
+        const allowedTypes = def.allowedTypes;
```

Comment:
question: won't this always be empty?

Reply:
Yes, I will use empty array instead here.